src/renderServer.ts

```diff
@@ -15,16 +15,14 @@ import {validateRenderData} from './validate';
 export function renderServer(config: ConfigService) {
   const app = express();
 
-  Sentry.getCurrentHub().bindClient(
-    new Sentry.NodeClient({
-      dsn: process.env.SENTRY_DSN,
-      integrations: [
-        new Integrations.Express({app}),
-        new Sentry.Integrations.Http({tracing: true}),
-      ],
-      tracesSampleRate: 1,
-    })
-  );
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    integrations: [
+      new Integrations.Express({app}),
+      new Sentry.Integrations.Http({tracing: true}),
+    ],
+    tracesSampleRate: 1,
+  });
```

Comment:
Just asking out of curiosity: Is there a reason you made the change from creating a client to calling `Sentry.init`?